crates/uv-build/src/lib.rs

```diff
@@ -331,6 +332,8 @@ pub struct SourceBuild {
     package_id: String,
     /// Whether we do a regular PEP 517 build or an PEP 660 editable build
     build_kind: BuildKind,
+    /// Environment variables to be passed in during metadata or wheel building
+    environment_variables: FxHashMap<String, String>,
```

Comment:
Do we want the keys to be `String` or `OsString`? This kinda depends where you get them from, but if it's from other os apis `OsString` would remove a potentially erroneous conversion.

Reply:
Very good idea!

Reply:
I changed this 👍